seed/analysis_pipelines/better.py

```diff
@@ -221,6 +223,8 @@ def _build_better_input(analysis_property_view, meters):
         errors.append('Linked PropertyState is missing a State')
     if property_state.gross_floor_area is None:
         errors.append('Linked PropertyState is missing a gross floor area')
+    if property_state.property_type is None:
+        errors.append('Linked PropertyState is missing a property type')
```

Comment:
Documenting our discussion, we should validate the user has provided one of the required property_types, otherwise BETTER 500s